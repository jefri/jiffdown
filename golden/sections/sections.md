# Chapter

Chapter overview

## Section 1

Section 1 lead

### Subsection 1.1

Subsection 1.1 details

### Subsection 1.2

Subsection 1.2 details

## Section 2

Section 2 lead

### Subsection 2.1

Subsection 1.1 details

### Subsection 2.2

Subsection 1.2 details
