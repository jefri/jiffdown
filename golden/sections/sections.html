<section>
<h1 id="chapter">Chapter</h1>
<p>Chapter overview</p>
<section>
<h2 id="section-1">Section 1</h2>
<p>Section 1 lead</p>
<section>
<h3 id="subsection-11">Subsection 1.1</h3>
<p>Subsection 1.1 details</p>
</section>
<section>
<h3 id="subsection-12">Subsection 1.2</h3>
<p>Subsection 1.2 details</p>
</section>
</section>
<section>
<h2 id="section-2">Section 2</h2>
<p>Section 2 lead</p>
<section>
<h3 id="subsection-21">Subsection 2.1</h3>
<p>Subsection 1.1 details</p>
</section>
<section>
<h3 id="subsection-22">Subsection 2.2</h3>
<p>Subsection 1.2 details</p>
</section>
</section>
</section>
